src/sdk/pynni/nni/nas/pytorch/fixed.py

```diff
@@ -73,6 +77,36 @@ def sample_final(self):
         """
         return self._fixed_arc
 
+    def replace_layer_choice(self, module=None, prefix=""):
+        """
+        Replace layer choices with selected candidates. It's done with best effort.
+        In case of weighted choices or multiple choices. if some of the choices on weighted with zero, delete them.
+        If single choice, replace the module with a normal module.
+
+        Parameters
+        ----------
+        module : nn.Module
+            Module to be processed.
+        prefix : str
+            Module name under global namespace.
+        """
+        if module is None:
+            module = self.model
+        for name, mutable in module.named_children():
+            global_name = (prefix + "." if prefix else "") + name
+            if isinstance(mutable, LayerChoice):
+                chosen = self._fixed_arc[mutable.key]
+                if sum(chosen) == 1 and max(chosen) == 1 and not mutable.return_mask:
```

Comment:
what is the meaning of `not mutable.return_mask`?

Reply:
If true, layer choice will return the choice vector. If we replace the layer choice module with the selected module, choice vector will be no longer available, and users’ code will crash.